### PR TITLE
SSAO

### PR DESCRIPTION
**Pull request by AtiX**
Bildschirmkoordiantensystemumgebungsverschattung!

### CODE REVIEW

**Comment by adius** [comment]
- [ ] Format comments like `# <UppercaseCharacter>`

File: src/client/rendering/Renderer.coffee [hunk 1]
```diff
@@ -93,6 +134,16 @@ class Renderer
 					@usePipelineFxaa = false
 					@pipelineRenderTarget = null
 
+			# determine wether to use SSAO
```
Comment (adius) [inline src/client/rendering/Renderer.coffee]: Unintuitive if else branching. Should be simplified

File: src/client/rendering/shader/ssaoBlurPart.coffee [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	getVertexInMain: ->
+		return''
+
+	getFragmentVariables: ->
+		return ''
+
+	getFragmentPreMain: ->
+		return '
+			float ssaoBlur(vec2 texCoords){
+				float dX = 1.0 / texWidth;\n
```
Comment (adius) [inline src/client/rendering/shader/ssaoBlurPart.coffee]: Why `\n` ?

File: src/client/rendering/shader/ssaoBlurPart.coffee [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+			}
+		'
+
+	getFragmentInMain: ->
+		return '
+			float blur = ssaoBlur(vUv);
+			col = vec4(0.0);
+			currentOpacity = 1.0 - blur;
+
+			/*col = texture2D(tColor, vUv);
```
Comment (adius) [inline src/client/rendering/shader/ssaoBlurPart.coffee]: Delete comment

File: src/plugins/legoBoard/LegoBoard.coffee [hunk 4]
```diff
@@ -26,7 +26,7 @@ module.exports = class LegoBoard
 		@_initMaterials()
 
 		#create baseplate
-		box = new THREE.BoxGeometry(400, 400, 8)
+		box = new THREE.BoxGeometry(400, 400, 8, 5, 5)
```
Comment (adius) [inline src/plugins/legoBoard/LegoBoard.coffee]: Create a comment to explain magic numbers

Comment (AtiX) [inline src/plugins/legoBoard/LegoBoard.coffee]: I don't think it is useful to explain a well documented constructor?

Comment (adius) [inline src/plugins/legoBoard/LegoBoard.coffee]: It sucks when you first have to look it up in the three documentation to understand anything

Comment (AtiX) [inline src/plugins/legoBoard/LegoBoard.coffee]: althogh I don't agree.. [x] fixed

**Comment by AtiX** [comment]
@adius :trumpet: Trööt trööt :)

**Pull request merged**